### PR TITLE
adds to README

### PR DESCRIPTION
**Pull request by joshuathayer**
Just clarifies and adds to the README.

### CODE REVIEW

**Comment by conorsch** [comment]
CI requires commits to be signed, so I'll rebase these shortly.

**Comment by conorsch** [comment]
Rebased with signed commits and pushed up. Added a few changes. @emkll the manual build steps _look_ accurate to me, can you confirm? Happy to run through an end-to-end build with those steps, but we can also refer to https://github.com/freedomofpress/securedrop-workstation/blob/cebaf303899e00b52fbe34e243092c6ac0bf0690/builder/build-workstation-template for reference. Mostly the ambiguity, to my eye, is around adjacent git repos and implicit `cd` operations.

File: README.md [hunk 1]
```diff
@@ -3,30 +3,54 @@
...
+
+0. Checkout the [securedrop workstation repo](https://github.com/freedomofpress/securedrop-workstation)
+1. Run `make template``
 
 ### Manual build
-```
+
 0. Clone the qubes-builder repository
-2. Copy the securedrop-workstation.conf  ../../builder.conf
+2. Copy the `securedrop-workstation.conf`  ../../builder.conf
```
Comment (emkll) [inline README.md]: For more clarity, propose changing to something like:

from `qubes-template-securedrop-workstation`, copy `securedrop-workstation.conf` as `builder.conf` in the qubes-builder repository root

Comment (conorsch) [reply README.md]: And then the remainder of the commands must be run from inside `qubes-builder` root, yes?

Comment (emkll) [reply README.md]: correct

Comment (conorsch) [reply README.md]: Great feedback, revised!

**Pull request merged**